charts/stable/anonaddy/questions.yaml

```diff
@@ -59,6 +54,11 @@ questions:
                                         schema:
                                           type: string
                                           default: "tls"
+                                          enum:
```

Comment:
Is this correct? Ui will display nothing but the value I’m the container will be none

Reply:
it was not.